pkg/controller/controller.go

```diff
@@ -48,24 +48,45 @@ func GenerateUUID() string {
 	s := uuid.String()
 	return s
 }
+// This function provides the kube client config file with the provided inputs
+func getConfig(configFile, kubeURL, kubeServAcctToken string) (*restclient.Config, error) {
```

Comment:
Somewhere i read we have to snake style for coding and i see most Go project starts function name with capital. Can we stick with some rules. You can suggest what is right and we can go head with that